tools/Azure.Mcp.Tools.Monitor/src/Commands/BaseMonitorCommand.cs

```diff
@@ -14,19 +15,18 @@ public abstract class BaseMonitorCommand<
     : SubscriptionCommand<TOptions>
     where TOptions : SubscriptionOptions, IWorkspaceOptions, new()
 {
-    private readonly Option<string> _workspaceOption = WorkspaceOptionDefinitions.Workspace;
-
     protected override void RegisterOptions(Command command)
     {
         base.RegisterOptions(command);
-        RequireResourceGroup();
-        command.Options.Add(_workspaceOption);
+        command.Options.Add(OptionDefinitions.Common.ResourceGroup.AsRequired());
+        command.Options.Add(WorkspaceOptionDefinitions.Workspace);
```

Comment:
It seems `BaseMonitorCommand` should drop this property, as this prevents `BaseMonitorHealthModelsCommand` from extending this, forcing duplication.